docs/.vuepress/config.js

```diff
@@ -1,55 +1,86 @@
 module.exports = {
-  title: "Shopware-PWA",
+  title: "PWA",
```

Comment:
we could leave it here as logo image has just "docs" text :)
```suggestion
  title: "shopware PWA",
```

![image](https://user-images.githubusercontent.com/13100280/81310844-ef1d9e80-9084-11ea-8ddf-e02534ecc57d.png)

Reply:
Fair point :)